react/features/recording/middleware.js

```diff
@@ -181,6 +182,8 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
                 if (soundID) {
                     dispatch(playSound(soundID));
                 }
+
+                APP.API.notifyRecordingStatusChanged(true, mode);
```

Comment:
@saghul is this ok for mobile?

Reply:
No, mobile doesn't have the APP global.

Reply:
I should be checking that typeof APP is not undefined, correct?
```javascript
if (typeof APP !== 'undefined')
```